src/sentry/south_migrations/0486_incident_type.py

```diff
@@ -0,0 +1,1407 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Adding field 'Incident.type'
+        db.add_column('sentry_incident', 'type',
```

Comment:
This will be fine to run. We have only 20 or so incident rows, so adding a new non-null column with a default won't take long